src/components/profile/editor-viewer.tsx

```diff
@@ -202,7 +202,7 @@ export const EditorViewer = <T extends Language>(props: Props<T>) => {
             fontFamily: `Fira Code, JetBrains Mono, Roboto Mono, "Source Code Pro", Consolas, Menlo, Monaco, monospace, "Courier New", "Apple Color Emoji"${
               getSystem() === "windows" ? ", twemoji mozilla" : ""
             }`,
-            fontLigatures: true, // 连字符
+            fontLigatures: false, // 连字符
```

Comment:
Probably not. We might want other ligatures (`=>`, `->`, etc.)

Block for now, waiting for further discussions.